cfpb/cfgov-refresh
# Remove email signup from staying afloat page

Remove the "Get email updates about our work for students" email signup box from the "Helping borrowers find ways to stay afloat" page.

## Removals

- Removed "Get email updates about our work for students" email signup box
- Also remove a stray `</style>` tag

## Testing

1. Visit http://localhost:8000/practitioner-resources/students/helping-borrowers-find-ways-to-stay-afloat/
2. Confirm the email signup box is gone

## Screenshots

Current | Changed
---- | ----
![current](https://user-images.githubusercontent.com/1862695/61740521-d42fa700-ad5c-11e9-876a-97a14b650127.png) | ![changed](https://user-images.githubusercontent.com/1862695/61740528-d7c32e00-ad5c-11e9-862c-b8310126dd37.png)

## Notes

- I kept the relevant styles around in `cfgov/legacy/static/nemo/_/c/styles.css` since I wasn't sure if they were used on other legacy pages

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
